core/modules/admin_bar/admin_bar.module

```diff
@@ -199,8 +200,7 @@ function admin_bar_preprocess_page(&$variables) {
   $args = explode('/', $_GET['q']);
   if ($args[0] == 'admin' && !empty($args[1])) {
     $settings['activeTrail'] = url($args[0] . '/' . $args[1]);
-  }
-  elseif (backdrop_is_front_page()) {
+  } elseif (backdrop_is_front_page()) {
```

Comment:
elseif should start in a new line. Please check the [coding standards for Backdrop CMS](https://docs.backdropcms.org/php-standards#control-structures).